### PR TITLE
Do not crash admin while listing partially shipped orders

### LINKED ISSUE #8717: You cannot visualize shipping_state = partially_shipped in admin dashboard
I have a order with property shipping_state = partially_shipped

I cannot visualize this order.

>Unable to find template "@SyliusAdmin/Order/Label/ShippingState/partially_shipped.html.twig" (looked into: /var/www/html/app/Resources/SyliusAdminBundle/views, /var/www/html/vendor/sylius/sylius/src/Sylius/Bundle/AdminBundle/Resources/views) in @SyliusAdmin/Order/Show/_shipments.html.twig at line 2.

Is it a bug ?

In folder https://github.com/Sylius/SyliusAdminBundle/tree/master/Resources/views/Order/Label/ShippingState

Only there are three templates, to states cancelled, ready, shipped...

### LINKED ISSUE #8719: Missing label for partially refunded payments
Similar to #8717, mentioned in https://github.com/Sylius/Sylius/pull/8718#issuecomment-331156262.

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #8717; fixes #8719  |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<span class="ui blue{% if attached is defined and attached == true %} top attached{% endif %} label">
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig]: `ready` state is blue, maybe different colour?

**Comment by GSadee** [comment]
There is also missing template for a `partially_refunded` payment state

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig (hunk 1 above)
Comment (pamil) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig]: No idea though, partially shipped is something between ready & shipped, blue & green, clock & plane; the most important thing here is not to crash admin interface, I'm open for colour / icon suggestions though :)

Comment (pamil) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig]: Like in payments, partially paid has the same colour as cancelled one.

**Comment by pamil** [comment]
@GSadee colours & icons adjusted 🎉

File: src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<span class="ui yellow{% if attached is defined and attached == true %} top attached{% endif %} label">
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig]: I know, I'm annoying and this is irrelevant, but `cancelled` state is yellow 😆

Comment (pamil) [reply src/Sylius/Bundle/AdminBundle/Resources/views/Order/Label/ShippingState/partially_shipped.html.twig]: Yeah, that's the same with partially paid and cancelled, which already exists for a while and nobody's complaining :D

**Comment by GSadee** [comment]
Thanks Kamil! 🌮

**Pull request merged**